FAQ.md

```diff
@@ -0,0 +1,111 @@
+# Frequently Asked Questions
+
+## Where can I find upgrades?
+https://origin-release.svc.ci.openshift.org/
+
+Each Preview/Beta version is marked with an asterix *
+
+> Bug: Take care that if you upgraded to a version which is a nightly, that they get   
+ deleted automatically after 48 hours from the CI registry. You could come 
+ into a situation where there is no tested upgrade path to a newer version. The removed images may result in pods stuck in ImagePullBackoff (because the pod tries to download an image from the CI registry which isn't available anymore).
+
+## How can I upgrade my cluster to a new version?
+Find a version where a tested upgrade path is available from your version for on 
+
+https://origin-release.svc.ci.openshift.org
+
+Upgrade options:
+
+**Preferred** ways:
+* Web Console: Home -> Overview -> Tab: Cluster, Card: Overview -> View settings -> Update Status
+
+  Currently (as of 2020-04-10) this doesn't work because the nightly builds are not signed. Will be resolved soon !
```

Comment:
Lets mention a tracking bug for that - https://github.com/openshift/okd/issues/138

Reply:
Done